### PR TITLE
Fix #2194: Updated Nearby ⓘ icon to include 'Search this area' feature

### LINKED ISSUE #2194: Update Nearby ⓘ icon content after "search this area" feature
**Summary:** 

We need to update nearby information box text and include search this area feature ( #2051 ) in it.

### PR DESCRIPTION
**Pull request by ShridharGoel**
**Description (required)**

Fixes #2194 

What changes did you make and why?

Updated content of dialog box which opens up on clicking the i icon on the Nearby tab.

**Tests performed (required)**

Tested betaDebug on API 25.

**Screenshots showing what changed**

screenshot_2018-12-20-23-27-29-954_fr free nrw commons beta

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -296,7 +296,7 @@
   <string name="retry">Retry</string>
 
   <string name="showcase_view_got_it_button">Got it!</string>
-  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles</string>
+  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles.\n\nClicking on \'SEARCH THIS AREA\' locks the map and launches a nearby search around that location.</string>
   <string name="showcase_view_list_icon">Tapping this button brings up a list of these places</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml]: Hmm, I think locking the map is something only developers should know. But I am not sure what kind of message would be better. Let's wait some ideas from theam.

Comment (ShridharGoel) [reply app/src/main/res/values/strings.xml]: What about "...fixes the map at that location..."? Or maybe we can omit that part and just write that it launches a nearby search around that location?

**Comment by domdomegg** [comment]
What's going on with this PR - have we agreed on some wording for this yet @neslihanturan?

**Comment by domdomegg** [comment]
I'm going to merge this for now given it's been here so long, if we need to change the wording we can open a new issue.

**Pull request merged**